src/index.js

```diff
@@ -57,9 +57,11 @@ class ServerlessThundraPlugin {
     constructor(serverless = {}, options) {
         this.serverless = serverless
         this.prefix =
+            ((this.serverless.service.custom || { thundra: {} }).thundra || {})
+                .package_json_path ||
             options.prefix ||
-            this.serverless.config.servicePath ||
-            process.env.npm_config_prefix
+            process.env.npm_config_prefix ||
```

Comment:
Good catch :+1:

Reply:
Given that this was basically being ignored (so no one is depending on it), should I take the opportunity and rename it `PACKAGE_JSON_PATH` ?

I can also rename the `--prefix` cli argument (since this is a bit too generic imo) to `--package-json-path` to have all options consitent. But this would then be a breaking change - not sure if that is ok. Or else i can accept the two options `--prefix` and `--package-json-prefix` and add a deprecation warning on the first one.